sdk/python/build.sh

```diff
@@ -24,36 +24,122 @@
 #   apt-get update -y
 #   apt-get install --no-install-recommends -y -q default-jdk
 #   wget http://central.maven.org/maven2/io/swagger/swagger-codegen-cli/2.4.1/swagger-codegen-cli-2.4.1.jar -O /tmp/swagger-codegen-cli.jar
+#   # where /tmp/swagger-codegen-cli.jar in the wget line above is the same as the variable SWAGGER_CODEGEN_CLI_FILE defined below
+
+
+set -e
+set -o nounset
+set -o pipefail
+
+usage()
+{
+    echo "Usage: build.sh
+     ( [-o, --output OUTPUT_PKG_FILE] | [-i, --install ] )
+     [-h help]
+
+    The -o flag and -i flag should not both be set at the same time.  No flags is equivalent to -o ./kfp.tar.gz
+
+    Examples:
+    build.sh                               # build and output a package, kfp.tar.gz, in this directory
+    build.sh --output ~/my_kfp_pgk.tar.gz  # build and output a package, my_kfp_pkg.tar.gz in your home dir
+    build.sh --install                     # install an editable copy of this kfp package in this directory
+    "
+}
+
+SWAGGER_CODEGEN_CLI_FILE=/tmp/swagger-codegen-cli.jar
+if [[ ! -f ${SWAGGER_CODEGEN_CLI_FILE} ]]; then
+    echo "You don't have swagger codegen installed.  See the comments on this file for install instructions."
```

Comment:
Maybe we should just install it then...

Reply:
i think if we're going to install them, then the install process should be more robust than this. that means automating install of java, setting java home, etc.  i think it's a good idea but out of scope for this update.

Reply:
I just thought that we already have that in a comment above, so why not put it here.
At least the wget swagger part.
No user will have that .jar in their `/tmp/`

Reply:
sure, added in.  it doesn't solve our overall install problem, but assuming they're on a system with wget, it'll download the swagger cli automatically.